packages/eds-core-react/src/components/Search/Search.tsx

```diff
@@ -117,6 +117,12 @@ const SearchInput = styled(Input)(({ theme, disabled }) => {
     &:focus {
       outline: none;
     }
+    &:-webkit-autofill {
```

Comment:
Did it work without the vendor prefix? I thingk `styled-components` has built in autoprefixer 🤔

Reply:
It does not. The non-prefixed alone does not work, and does not get autoprefixed, so this is more of a future-proofing (css-tricks recommmended including it)